src/display/api.js

```diff
@@ -201,6 +201,9 @@ if (typeof PDFJSDev === "undefined" || !PDFJSDev.test("PRODUCTION")) {
  *   `OffscreenCanvas` in the worker. Primarily used to improve performance of
  *   image conversion/rendering.
  *   The default value is `true` in web environments and `false` in Node.js.
+ * @property {boolean} [canvasMaxAreaInBytes] - The integer value is used to
```

Comment:
Sorry @calixteman , wasn't that supposed to be 

```suggestion
 * @property {number} [canvasMaxAreaInBytes] - The integer value is used to
```

?

Reply:
Yep it should. Would you mind to make a PR please ?

Reply:
Already did!